pom.xml

```diff
@@ -122,6 +123,12 @@
             <artifactId>hadoop-common</artifactId>
             <version>${hadoop.two.version}</version>
             <scope>provided</scope>
+            <exclusions>
+              <exclusion>
+                <groupId>com.google.protobuf</groupId>
```

Comment:
Why to exclude this if `hadoop-common` is already `provided` dependency?

Reply:
Provided still affects compile time version resolution and it crashes.